src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -250,7 +251,9 @@ void func_80BA39C8(EnToto* this, GlobalContext* globalCtx) {
         return;
     }
 
-    if ((globalCtx->sceneNum == SCENE_MILK_BAR && !(gSaveContext.time >= 0x4000 && gSaveContext.time < 0xED02)) ||
+    //! @TODO: 0xED02 nor 0xED01 match CLOCK_TIME macro
+    if ((globalCtx->sceneNum == SCENE_MILK_BAR &&
+         !(gSaveContext.time >= CLOCK_TIME(6, 0) && gSaveContext.time < 0xED02)) ||
```

Comment:
I assume the second one can't be written nicely?

Reply:
No, that is what the TODO is for. The thing 0xED02 isn't even close to either boundary.
CLOCK_TIME(22, 13) = 0xECFA  and CLOCK_TIME(22, 14) = 0xED27

Reply:
Wow, that's annoying. Wonder if it's more sensible with the mult and div the other way or something.